test/jdk/javax/swing/JColorChooser/Test4222508.java

```diff
@@ -21,36 +21,47 @@
  * questions.
  */
 
+import java.awt.BorderLayout;
+import javax.swing.JCheckBox;
+import javax.swing.JColorChooser;
+import javax.swing.JFrame;
+
 /*
  * @test
  * @bug 4222508
+ * @library /test/jdk/java/awt/regtesthelpers /test/lib
+ * @build PassFailJFrame
  * @summary Tests the color chooser disabling
- * @author Sergey Malenkov
- * @run applet/manual=yesno Test4222508.html
+ * @run main/manual Test4222508
  */
+public final class Test4222508 {
+    public static void main(String[] args) throws Exception {
+        PassFailJFrame.builder()
+                .title("Test4222508")
+                .instructions("Click on colors in the JColorChooser. Then uncheck the checkbox \n" +
+                        "and click on colors again. \n" +
+                        "If the JColorChooser is disabled when the checkbox is unchecked, \n" +
+                        "then pass the test.")
```

Comment:
The line breaks here are in random odd places.
It should be more like this
"Click on colors in the JColorChooser.\n" +
"Then uncheck the checkbox and click on colors again.\n" +
"If the JColorChooser is disabled when the checkbox is unchecked, then pass the test.";

You can also  use text blocks if you want